deploy/Makefile

```
@@ -0,0 +1,75 @@
+
+.PHONY: status
+status: ## Rollout status + pods
+	kubectl -n $(NAMESPACE) rollout status deploy/$(RELEASE) --timeout=5m || true
+	kubectl -n $(NAMESPACE) get pods,svc,pvc
+
+.PHONY: logs
+logs: ## Tail omni-api logs
+	kubectl -n $(NAMESPACE) logs -l app.kubernetes.io/name=omni,app.kubernetes.io/component=api -f --tail=200
+
+.PHONY: health
+health: ## Port-forward + curl /health
```

checkmake (0.3.2)

[warning] Target body for "health" exceeds allowed length of 5 lines (6).

(maxbodylength)